habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py

```diff
@@ -430,83 +435,91 @@ def add_cat_fields_to_episodes(
     if num_episodes == -1:
         num_episodes = len(episodes["episodes"])
 
-    for episode in tqdm(episodes["episodes"]):
-        scene_id = episode["scene_id"]
-        sim = initialize_sim(
-            sim,
-            existing_rigid_objects,
-            scene_id,
-            episode["scene_dataset_config"],
-            episode["additional_obj_config_paths"],
-            debug_viz,
-        )
-        load_navmesh(sim, scene_id)
-
-        rom = sim.get_rigid_object_manager()
-        existing_rigid_objects = set(rom.get_object_handles())
-
-        rec = find_receptacles(sim)
-        rec_viewpoints, _ = load_receptacle_cache(
-            scene_id, rec_cache_dir, receptacle_cache
-        )[scene_id]
-        rec = [r for r in rec if r.parent_object_handle in rec_viewpoints]
-        rec_to_parent_obj = {r.name: r.parent_object_handle for r in rec}
-        obj_idx_to_name = load_objects(
-            sim, episode["rigid_objs"], episode["additional_obj_config_paths"]
-        )
-        populate_semantic_graph(sim)
-        all_rec_goals = collect_receptacle_goals(
-            sim, rec, rec_category_mapping=rec_category_mapping
-        )
-        obj_cat, start_rec_cat, goal_rec_cat = get_obj_rec_cat_in_eps(
-            episode,
-            obj_category_mapping=obj_category_mapping,
-            rec_category_mapping=rec_category_mapping,
-        )
+    with tqdm(total=num_episodes) as pbar:
+        for episode in episodes["episodes"]:
+            scene_id = episode["scene_id"]
+            episode["additional_obj_config_paths"] = ['data/objects/amazon_berkeley/configs/', 'data/objects/google_scanned/configs/', 'data/objects/ai2thorhab/configs/objects', 'data/objects/floorplanner/configs/objects']
```

Comment:
Seeing this reminds me to ask you: Are you using YCB in the val set?

Reply:
I'm not. Thanks for adding a comment here, this code shouldn't be here. I've removed it now.